README.md

```diff
@@ -1,11 +1,154 @@
 # CoreFx-TestData
 
-This repository contains test assets that are binary file or too large to be checked into corefx directly. Packages are produced and uploaded to the dotnet blob feed (https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
+This repository contains test assets that are binary files or too large to be checked into corefx directly. Packages are produced and uploaded to the dotnet blob feed: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json.
 
 Uploading the packages is currently a manual step. Community members can submit PRs but the deployment needs to be done by a .NET team member as described below.
 
 ## Workflow
-1. Make changes in the respective project and increments its version (usually just the patch fragment)
-2. Run `dotnet pack` on the project and find the produced nuget package in the project's bin folder.
-3. Use this build definition to upload the package to the dotnet blob feed: https://github.com/dotnet/core-eng/tree/master/Documentation/Tools/dotnet-core-push-oneoff-package. Make sure to clear the myget feed in the queue time variable section.
-4. Consume the updated package in corefx by incrementing the `PackageReference` version attribute: https://github.com/dotnet/corefx/blob/8d79b6117e9d584eebb8b6933bba83dd514010ca/external/test-runtime/XUnit.Runtime.depproj#L47-L54
+
+1. Modify the \*.csproj project file(s) and increment the version number.
+2. Generate the nuget package file for your new test data using `dotnet pack`.
+3. Test that corefx can import the new nuget package file.
+4. Verify that you can consume the new test assets in your corefx unit tests.
+5. Submit a corefx-testdata PR with your new test assets and bumped version project file.
+6. Upload the nuget package file to the dotnet blob feed.
+7. Submit a corefx PR with your unit test changes that consume the new nuget package.
+
+
+## Example
+
+We are going to make the following assumptions:
+
+- We cloned the `dotnet/corefx-testdata` GitHub project in `D:\corefx-testdata`.
+- We cloned the `dotnet/corefx` GitHub project in `D:\corefx`.
+- We are working on adding a new unit test for the GZip feature from the `System.IO.Compression` namespace, and the test depends on a file called `example.gz`.
```

Comment:
In order for your example to work, there is one more assumption that you must make, which is that people have run build.cmd in the corefx repo. If you don't do that, then you won't be able to run step 3.

Reply:
Right, I forgot to mention that. Thanks.
```suggestion
- We are working on adding a new unit test for the GZip feature from the `System.IO.Compression` namespace, and the test depends on a file called `example.gz`.
- We already compiled corefx with `build.cmd`.
```